src/clients/firewall_stage1_finish.rb

```diff
@@ -80,6 +88,77 @@ def main
       Builtins.y2milestone("firewall_stage1_finish finished")
       deep_copy(@ret)
     end
+
+  private
+
+    # Returns true if all services are known to firewall
+    # @param [Array <String>] services
+    # @return [Boolean] if all are known
+    def known_firewall_services?(services)
+      @all_known_services ||= SuSEFirewallServices.all_services.keys
+
+      (services - @all_known_services).empty?
+    end
+
+    # Convenience method for opening the ssh port during the first stage when
+    # enabled
+    def open_ssh_port(open)
+      # Open or close FW ports depending on user decision
```

Comment:
What about moving this text to the method's description? Additionally, documenting what `open` (a boolean) is would be nice.